src/logging.rs

```diff
@@ -0,0 +1,63 @@
+use command::Command;
+use error::Error;
+use slog::{Drain, Level, Logger};
+use slog_async::Async;
+use slog_term::{FullFormat, PlainDecorator};
+use std::fs::OpenOptions;
+use std::path::PathBuf;
+
+/// Create the logger for wasm-pack that will output any info warning or errors we encounter
+pub fn create_logger(cmd: &Command, verbosity: u8) -> Result<Logger, Error> {
```

Comment:
let's call this `logger.rs` and then have this function called new so we have `logger::new`